record-and-playback/core/lib/recordandplayback/edl/audio.rb

```diff
@@ -155,6 +185,7 @@ def self.render(edl, output_basename)
           if audioinfo[input[:filename]][:format][:format_name] == 'wav'
             ffmpeg_cmd += ['-ignore_length', '1']
           end
+          screenshare ? ffmpeg_cmd += ['-c:a', 'libopus'] : nil
```

Comment:
Why are you using the `libopus` decoder here instead of ffmpeg's builtin opus decoder? Have you had any specific issues with the builtin decoder causing problems?

Reply:
Yes. Higher frequency of A/V de-sync occurrences and a bit worse handling of lossy raw streams.
Don't have any data on that anymore, though.

Reply:
Can you confirm whether libopus is enabled on the ffmpeg package 2.3 is using?

Reply:
If you're having problems like that with the internal opus decoder, we should probably switch decoding all opus to use libopus. In any case, selecting the libopus decoder needs to be made conditional on the audioinfo saying that the codec is 'opus'.

I added the libopus decoder to the ffmpeg build a while back, search for "Enable decoders" in the build log to see what's there: https://launchpadlibrarian.net/500061740/buildlog_ubuntu-bionic-amd64.ffmpeg_7%3A4.2.4-1ubuntu0.1bbb2~18.04_BUILDING.txt.gz